guides/common/modules/proc_performing-discovery-in-pxeless-mode.adoc

```diff
@@ -0,0 +1,56 @@
+[id="performing-discovery-in-pxeless-mode_{context}"]
+= Performing Discovery in PXE-less mode
+[[Implementing_PXE_less_Discovery_{context}]]
+
+Discovery in PXE-less mode uses the Discovery ISO and requires you to attend to the process.
```

Comment:
```suggestion
Discovery in PXE-less mode uses the Discovery ISO and requires you to pay attention to the process.
```

Reply:
What's wrong with "attend"? When it's attended, they don't only pay attention, they have to perform some steps manually. How about "requires your manual intervention" then?